packages/wasm/src/core/polkadot-utils.js

```diff
@@ -20,6 +18,19 @@ const polkadotTypesToKeys = {
   ecdsa: 'EcdsaSecp256k1VerificationKey2019',
 };
 
+export function getKeyPairType(key) {
+  const keyType = key.type || key.constructor.VerKeyType;
+  if (keyType) {
+    return keyType;
+  }
+
+  if (key instanceof Ed25519Keypair) {
```

Comment:
we don't need to worry about BBS and BLS key types here?

Reply:
this is only used to generate dids, so I think it should not be needed. It also matches with certs implementation